.github/workflows/pull-request.yaml

```diff
@@ -110,6 +110,8 @@ jobs:
       - name: Unit Test Golang
         run: make test-coverage
         timeout-minutes: 30
+        env:
+          CI: true
```

Comment:
Alternatively, we could use the `GITHUB_ACTIONS` env var for this, which is always set to true in GH Actions runners.

https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
Thanks. I was looking for something like that but haven't looked hard enough, it seems 😄 I'll change it.